requirements/requirements.txt

```diff
@@ -39,7 +39,7 @@ pytz==2014.10
 PyYAML==3.10
 raven==3.1.17
 requests==2.0.0
-south==0.7.3
+south==1.0.2
```

Comment:
I tested this on staging and was able to deploy without any issues. Also reviewed the change log and didn't see anything that should break (so did @emord: https://github.com/dimagi/commcare-hq/pull/6794/files#r30916913)